mono/metadata/icall.c

```diff
@@ -312,18 +312,20 @@ array_set_value_impl (MonoArrayHandle arr_handle, MonoObjectHandle value_handle,
 
 #ifdef ENABLE_NETCORE
 #define WIDENING_MSG NULL
```

Comment:
Suggest local variable instead of macros.

Reply:
WIDENING_MSG is then used in a different macro